django_mongodb_extensions/debug_toolbar/panels/mql/tracking.py

```diff
@@ -26,7 +26,7 @@ class DebugToolbarWrapper(OperationDebugWrapper):
 
     def __init__(self, db, collection, logger):
         super().__init__(db, collection)
-        self.collection_name = collection.name
+        self.collection_name = f"{collection.name}."
```

Comment:
You should remove this line. These lines in `super()` already do the right thing:
```
use_collection = collection is not None
self.collection_name = f"{collection.name}." if use_collection else ""
```